src/components/pages/order/Main/Menu.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import styled from "styled-components";
+import { theme } from "../../../../theme";
+import { fakeMenu2 } from "../../../../fakeData/fakeMenu";
+import Card from "../../../reusable/Card";
+import { formatPrice } from "../../../../utils/maths";
+
+export default function Menu() {
+  const [menu, setMenu] = React.useState(fakeMenu2);
+
+  return (
+    <MenuStyled>
+      {menu.map(({ id, imageSource, title, price }) => {
+        return (
+          <Card
+            key={id}
+            imageSource={imageSource}
+            title={title}
+            leftDescription={formatPrice(price)}
+          />
+        );
+      })}
+    </MenuStyled>
+  );
+}
+
+const MenuStyled = styled.div`
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);

```

Comment:
Ici tu vas avoir un problème, essaie de resize t'as page horizontalement, tu verras que tes cards vont "disparaître" sur le côté, on souhaiterait plutôt que  les cards s'alignent du coup verticalement si il y a moins d'espace.

```suggestion
grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
```

Reply:
En effet, j'avais jamais eu le souci car ma width était fixe et pas en VW/VH